models/comments-model.js

```diff
@@ -32,3 +32,16 @@ exports.insertComment = (article_id, { username, body }) => {
       return data.rows[0];
     });
 };
+
+exports.removeCommentByID = (comment_id) => {
+  return db
+    .query("SELECT * FROM comments WHERE  comment_id = $1", [comment_id])
```

Comment:
This is absolutely fine, but also just to let you know, when doing a delete, if you check the rows, you can determine whether anything was deleted or not. If nothing was deleted, then you can assume the ID does not exist and you can throw the 404 error.